LayoutTests/imported/w3c/web-platform-tests/css/selectors/dir-pseudo-on-input-element.html

```diff
@@ -129,6 +129,30 @@
     }, `input element whose type attribute is in the ${type} state`);
 }
 
+test(() => {
```

Comment:
Maybe this test case should go at the bottom of the file? The expected output seems a little strange with the unique assertion in the middle.

Reply:
This was just imported from upstream, see https://chromium-review.googlesource.com/c/chromium/src/+/4851878

Reply:
I see, shouldn't we be re-syncing separately?

Reply:
We could, though it's more convenient in this case. Easier to integrate to the branch as well.